source/draw/RasterCanvas.ooc

```diff
@@ -56,11 +56,11 @@ RasterCanvas: abstract class extends Canvas {
 			}
 		}
 	}
-	drawLines: override func ~explicit (lines: VectorList<FloatPoint2D>, pen: Pen) {
-		if (lines count > 1)
-			for (i in 0 .. lines count - 1) {
-				start := IntPoint2D new(lines[i] x, lines[i] y)
-				end := IntPoint2D new(lines[i + 1] x, lines[i + 1] y)
+	drawLines: override func ~explicit (pointList: VectorList<FloatPoint2D>, pen: Pen) {
+		if (pointList count > 1)
```

Comment:
Not introduced here, but since it's all about cleaning up the `draw` namespace - this if statement is not needed. If `pointList count` is 1 or less, the for loop will not iterate.